index.html

```diff
@@ -6,43 +6,178 @@
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+          <div class="work-card">
+            <div class="title">Profesional Art Printing Data</div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+            <ul>
+              <li>
+                <div class="work-cart">html</div>
+              </li>
+              <li>
+                <div class="work-cart">bootstrap</div>
+              </li>
+              <li>
+                <div class="work-cart">Ruby</div>
+              </li>
+            </ul>
+            <div class="button">See Project</div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="top">
+          <span>About me</span>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="button">Get My Resume</div>
+        </div>
+        <hr />
+        <div class="bottom">
+          <div class="language">
+            <img src="./assets/losange.png" />
+            Language
+          </div>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+          <div class="frameworks">
+            <img src="./assets/Rectangle.png" />Frameworks
+          </div>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+          <div class="skills"><img src="./assets/cicle.png" />Skills</div>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
```

Comment:
- [ ] [REQUIRED] The technology categories ("Languages", "Frameworks" and "Skills") should all be in a `ul`, each of them in a `li` and the same goes for the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also in a list `ul` and each of them a `li`. Essentially, you will have a nested `ul` as shown below: [Read about list-nesting here](https://www.w3schools.com/html/tryit.asp?filename=tryhtml_lists_nested).
```
<ul>
    <li>languages
        <ul>          
            <li>Javascript</li>
            <li>CSS</li>
           <li>HTML</li>
        </ul> 
    </li>
    <li>Frameworks
          <ul>     
             <li>....</li>
            <li>...</li>
           <li>...</li>
        </ul> 
    </li>
    <li>Skills
         And so on
    </li>    
</ul>
```